worlds/LauncherComponents.py

```diff
@@ -72,9 +77,9 @@ def __call__(self, path: str):
         return False
 
 
-def launch_textclient():
+def launch_textclient(*args):
     import CommonClient
-    launch_subprocess(CommonClient.run_as_textclient, name="TextClient")
+    launch_subprocess(CommonClient.run_as_textclient(*args), name="TextClient")
```

Comment:
This looks like it's 1st calling `CommonClient.run_as_textclient`, then after that function returns `None`, that `None` is then passed as the 1st parameter to `launch_subprocess`, which launches a subprocess that does nothing, because `None` was passed as the target.

Is that right?